flightsql/src/main/java/io/deephaven/server/flightsql/FlightSqlResolver.java

```diff
@@ -0,0 +1,1902 @@
+            case COMMAND_GET_DB_SCHEMAS_TYPE_URL:
+                return CommandGetDbSchemasConstants.HANDLER;
+            case COMMAND_GET_PRIMARY_KEYS_TYPE_URL:
+                return commandGetPrimaryKeysHandler;
+            case COMMAND_GET_IMPORTED_KEYS_TYPE_URL:
+                return commandGetImportedKeysHandler;
+            case COMMAND_GET_EXPORTED_KEYS_TYPE_URL:
+                return commandGetExportedKeysHandler;
+            case COMMAND_GET_SQL_INFO_TYPE_URL:
+                return GET_SQL_INFO_HANDLER;
+            case COMMAND_STATEMENT_UPDATE_TYPE_URL:
+                return STATEMENT_UPDATE_HANDLER;
+            case COMMAND_GET_CROSS_REFERENCE_TYPE_URL:
+                return GET_CROSS_REFERENCE_HANDLER;
+            case COMMAND_STATEMENT_SUBSTRAIT_PLAN_TYPE_URL:
+                return STATEMENT_SUBSTRAIT_PLAN_HANDLER;
+            case COMMAND_PREPARED_STATEMENT_UPDATE_TYPE_URL:
+                return PREPARED_STATEMENT_UPDATE_HANDLER;
+            case COMMAND_GET_XDBC_TYPE_INFO_TYPE_URL:
+                return GET_XDBC_TYPE_INFO_HANDLER;
+        }
+        throw error(Code.UNIMPLEMENTED, String.format("command '%s' is unknown", typeUrl));
+    }
+
+    private TicketHandler ticketHandlerForResolve(SessionState session, Any message) {
+        final String typeUrl = message.getTypeUrl();
+        if (TICKET_STATEMENT_QUERY_TYPE_URL.equals(typeUrl)) {
+            final TicketStatementQuery ticketStatementQuery = unpackOrThrow(message, TicketStatementQuery.class);
+            final TicketHandler ticketHandler = queries.get(ticketStatementQuery.getStatementHandle());
+            if (ticketHandler == null) {
+                throw error(Code.NOT_FOUND,
+                        "Unable to find FlightSQL query. FlightSQL tickets should be resolved promptly and resolved at most once.");
+            }
+            if (!ticketHandler.isOwner(session)) {
+                // We should not be concerned about returning "NOT_FOUND" here; the handleId is sufficiently random that
+                // it is much more likely an authentication setup issue.
+                throw permissionDeniedWithHelpfulMessage();
+            }
+            return ticketHandler;
+        }
+        final CommandHandler commandHandler = commandHandler(session, typeUrl, false);
+        try {
+            return commandHandler.initialize(message);
+        } catch (StatusRuntimeException e) {
+            // This should not happen with well-behaved clients; or it means there is a bug in our command/ticket logic
+            throw error(Code.INVALID_ARGUMENT,
+                    "Invalid ticket; please ensure client is using an opaque ticket", e);
+        }
+    }
+
+    private Table executeSqlQuery(SessionState session, String sql) {
+        // See SQLTODO(catalog-reader-implementation)
+        // final QueryScope queryScope = sessionState.getExecutionContext().getQueryScope();
```

Comment:
this commented out line appears to be extraneous (same line is right below and not commented out).